src/SlotMachine.css

```diff
@@ -0,0 +1,123 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  padding: 50px;
+  background: linear-gradient(
+    -30deg,
+    rgb(80, 60, 60) 40%,
+    rgb(255, 255, 255) 100%
+  );
+}
+
+.slot-machine {
```

Comment:
here's a challenge for you. Can you use style-components to wrap these styles into the component?

https://styled-components.com/